lib/model/book.dart

```diff
@@ -0,0 +1,35 @@
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class Book {
+  final String id;
+  String title;
+  String cover;
+  final DateTime createdAt;
+  DateTime modifiedAt;
```

Comment:
Why are some fields final and others not?

Reply:
To be honest I just follow the post to see how he did it.  I guess final fields is those fields we don't want to modify later.

Reply:
Alright. Sounds good